example/vm/config.yaml

```diff
@@ -0,0 +1,201 @@
+        # spans.  Note that our current recommendation for exporter
+        # settings does not respond with back-pressure to the batch
+        # processor.  Due to exporter settings, this collector
+        # configuration will drop data when the ServiceNow service is
+        # (intentionally or accidentally) refusing data, instead of
+        # applying pressure backward, discussed in the `exporters`
+        # section.
+        send_batch_size: 1000
+        timeout: 1s
+        send_batch_max_size: 1500
+
+exporters:
+    otlp/ls:
+        endpoint: ingest.lightstep.com:443 # US data center
+        # endpoint: ingest.eu.lightstep.com:443 # EU data center
+        headers:
+            lightstep-access-token: "${LIGHTSTEP_ACCESS_TOKEN}"
+        # While we expect latency under one second, typically, we
+        # recommend a longer timeout than the default.
+        timeout: 5s
+        # Queue settings are required.  It does not make sense to use
+        # the exporter without a queue, it has to do with
+        # requiring the "num_consumers" limit configured in this
+        # section (i.e., we require a queue in order to limit the
+        # number of concurrent exports).
+        #
+        # Note that the queue settings are applied in unit-terms
+        # produced by the batch processor, so a number like 100 means
+        # the queue has support for 100 pre-batched items.  With up to
+        # 1500 spans each (from the batch processor), this
+        # configuration allows 150,000 spans to occupy memory.
+        sending_queue:
+            enabled: true
+            num_consumers: 4
+            queue_size: 100
+        # Retry settings are optional.
+        #
+        # Note that while retries are attempted, this component will
+        # begin to drop arriving data if the queue is not large
+        # enough.
+        retry_on_failure:
+            # We recommend disabling retries, since while the export is
+            # blocked it is likely that arriving spans will drop, and
+            # Otherwise, collectors will need substantial additional
+            # memory to survive transient failures.  Nevertheless, we
+            # recommend a limited retry policy to gracefully occasional
+            # failures, paired with a modest queue size.
+            #
+            # Note there is a persistent storage option inherited from a
+            # common collector component.  When persistent storage is
+            # configured, the default retry configuration is sensible.
+            #
+            # For more details on retry and queue settings, please refer to
+            # https://github.com/open-telemetry/opentelemetry-collector/blob/main/exporter/exporterhelper/README.md
+            enabled: true
+            max_elapsed_time: 30s
+    debug:
+        verbosity: basic
+        sampling_initial: 5
+        sampling_thereafter: 200
+
+extensions:
+    # The healthcheck extension provides an endpoint that provides
```

Comment:
remove second "provides". 
i.e. ...an endpoint that supports liveness checks...